adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -96,7 +95,7 @@ void HybrisProximityAdaptor::sendInitialData()
                ProximityData *d = buffer->nextSlot();
                d->timestamp_ = Utils::getTimeStamp();
                d->withinProximity_ = near;
-               d->value_ = near ? 10 : 0;
+               d->value_ = absinfo.value;
```

Comment:
In the file-could-not-be-opened-else there is also lastNearValue=-1 -> withinProximity=true, value = 10;